services/ui-src/src/components/cards/EvaluationPlanEntity.test.tsx

```diff
@@ -0,0 +1,31 @@
+import { render, screen } from "@testing-library/react";
+import { testA11y } from "utils/testing/commonTests";
+import { mockEvaluationPlanFormattedEntityData } from "utils/testing/mockEntities";
+import { EvaluationPlanEntity } from "./EvaluationPlanEntity";
+
+const evaluationPlanEntity = (
+  <EvaluationPlanEntity
+    formattedEntityData={mockEvaluationPlanFormattedEntityData}
+  />
+);
+
+describe("<EvaluationPlanEntity />", () => {
+  beforeEach(() => {
+    render(evaluationPlanEntity);
```

Comment:
Having `render(evaluationPlanEntity);` here means it'll double render in `testA11y` which may not be a problem here but causes problems in some other tests. If you have only one test, you don't really need the `beforeEach` block and can do this instead:
```
test("EvaluationPlanEntity renders correctly", () => {
  render(evaluationPlanEntity);
  expect(
    screen.getByRole("heading", { name: "mockEvaluationPlanName" })
  ).toBeVisible();
});
```

Reply:
Gotcha. I do plan to add more tests to these entity components so I'll be making updates.